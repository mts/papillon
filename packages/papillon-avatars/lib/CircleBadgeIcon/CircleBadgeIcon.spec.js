import {
  circleBadgeSmallIcon,
  circleBadgeMediumIcon,
  circleBadgeLargeIcon,
} from './CircleBadgeIcon.render'

describe('<Avatar />', () => {
  describe('Snaphot', () => {
    test('must match circleBadgeSmallIcon', () => {
      expect(global.renderToJSON(circleBadgeSmallIcon)).toMatchSnapshot()
    })

    test('must match circleBadgeMediumIcon', () => {
      expect(global.renderToJSON(circleBadgeMediumIcon)).toMatchSnapshot()
    })

    test('must match circleBadgeLargeIcon', () => {
      expect(global.renderToJSON(circleBadgeLargeIcon)).toMatchSnapshot()
    })
  })
})
