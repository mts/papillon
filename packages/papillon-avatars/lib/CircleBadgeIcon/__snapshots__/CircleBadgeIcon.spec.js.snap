// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Avatar /> Snaphot must match circleBadgeLargeIcon 1`] = `
<a
  className="CircleBadge CircleBadge--large bg-gray-dark"
  href="href"
  title="title"
>
  <svg
    aria-hidden="true"
    className="CircleBadge-icon text-white"
    height={16}
    role="img"
    style={
      Object {
        "display": "inline-block",
        "fill": "currentColor",
        "userSelect": "none",
        "verticalAlign": "text-bottom",
      }
    }
    viewBox="0 0 10 16"
    width={10}
  >
    <path
      d="M10 7H6l3-7-9 9h4l-3 7 9-9z"
      fillRule="evenodd"
    />
  </svg>
</a>
`;

exports[`<Avatar /> Snaphot must match circleBadgeMediumIcon 1`] = `
<a
  className="CircleBadge CircleBadge--medium bg-gray-dark"
  href="href"
  title="title"
>
  <svg
    aria-hidden="true"
    className="CircleBadge-icon text-white"
    height={16}
    role="img"
    style={
      Object {
        "display": "inline-block",
        "fill": "currentColor",
        "userSelect": "none",
        "verticalAlign": "text-bottom",
      }
    }
    viewBox="0 0 10 16"
    width={10}
  >
    <path
      d="M10 7H6l3-7-9 9h4l-3 7 9-9z"
      fillRule="evenodd"
    />
  </svg>
</a>
`;

exports[`<Avatar /> Snaphot must match circleBadgeSmallIcon 1`] = `
<a
  className="CircleBadge CircleBadge--small bg-yellow"
  href="href"
  title="title"
>
  <svg
    aria-hidden="true"
    className="CircleBadge-icon text-white"
    height={16}
    role="img"
    style={
      Object {
        "display": "inline-block",
        "fill": "currentColor",
        "userSelect": "none",
        "verticalAlign": "text-bottom",
      }
    }
    viewBox="0 0 10 16"
    width={10}
  >
    <path
      d="M10 7H6l3-7-9 9h4l-3 7 9-9z"
      fillRule="evenodd"
    />
  </svg>
</a>
`;
